src/mlpack/core/util/program_doc.cpp

```diff
@@ -1,8 +1,9 @@
 /**
  * @file core/util/program_doc.cpp
+ * @author Yashwant Singh Parihar
  * @author Ryan Curtin
  *
- * Implementation of the ProgramDoc class.  The class registers itself with IO
+ * Implementation of the mutiple classes.  The classes registers itself with IO
  * when constructed.
```

Comment:
This might be a little clearer:
```suggestion
 * Implementation of mutiple classes that store information related to a binding.
 * The classes register themselves with IO when constructed.
```